src/Headroom.tsx

```diff
@@ -110,8 +108,8 @@ class Headroom extends React.PureComponent<PropsType, StateType> {
     }
     if (window.pageYOffset !== undefined) {
       return window.pageYOffset
-    } else if (window.scrollTop !== undefined) {
-      return window.scrollTop
+    } else if (window.scrollY !== undefined) {
+      return window.scrollY
```

Comment:
This is the only place I actually had to change the content of the code. Apparently window does not have the property `scrollTop` so I guessed `scrollY` would be the equivalent.

Reply:
You mean to remove the complete if/else if/else block and only keep the error? Just making sure we're on the same page :)